.grabl/automation.yml

```diff
@@ -64,20 +64,20 @@ build:
         export ARTIFACT_PASSWORD=$REPO_VATICLE_PASSWORD
```

Comment:
Temporarily comment these tests while they are failing due to unreleased Cluster